### ISSUE TITLE
About prediction use BBox regression and SVM classification

### ISSUE THREAD

**Issue by Scrooge20291224** [1297rohit/RCNN#4]
I am not sure, but I guess you are using several different (compared to the paper of RCNN) methods to train and forward the model, maybe you are doing this for making it easy to be understand by us, and I am not sure so I want to point out my guesses to verified it, I will appreciate it if you could reply my guess:
1.use a softmax instead of SVM to make classification
2.I haven't seen bbox regression during training, (maybe I missed...?) and since your code is my first try on R-CNN, I guess the origin model (I mean the one from the paper) use its own bbox regression to get coordinates of box for features-extraction and classification...?
Anyway, your codes help me a lot, Thx!

**Comment by 1297rohit** [comment]
Hello,
Yes you are right this implementation is slightly different from the implementation in paper. I have used softmax for classification and I have used selective search algorithm to identify ROI in image and calculated IOU over it. I have explained the full approach in my article. You can read it for better understanding. Link : https://towardsdatascience.com/step-by-step-r-cnn-implementation-from-scratch-in-python-e97101ccde55
